### PR TITLE
enh: Show more informative buttons when selecting CSV file location

### PR DESCRIPTION
**Pull request by susnux**
* Show visual error toast to user when there was something not working instead of only showing a browser console debug message
* Some general refactoring
* Show more informative buttons, see below

### before

https://github.com/nextcloud/forms/assets/1855448/2d1cae99-fd8b-4c04-8b94-9c81ef98164e

### after

https://github.com/nextcloud/forms/assets/1855448/45c699da-5472-4d48-8fac-e236dfc64f0c

### CODE REVIEW

File: src/views/Results.vue [hunk 1]
```diff
@@ -254,20 +250,53 @@ export default {
...
+			try {
+				await this.picker.pick()
+			} catch (error) {
+				// User aborted
+				logger.debug('No file selected', { error })
+			}
+		},
+
+		/**
+		 * @param {string|undefined} path
```
Comment (Chartman123) [inline src/views/Results.vue:282]: Could you please add the description here, so that the warning disappears? 🙂

**Review by Chartman123 — APPROVED** [review]
Just the little comment about the JSDoc description, rest seems fine 👍🏻

**Review by jancborchardt — APPROVED** [review]
Looks gopd design-wise :)

File: src/views/Results.vue (hunk 1 above)
Comment (susnux) [reply src/views/Results.vue]: Done :heavy_check_mark:

**Review by Chartman123 — APPROVED** [review]

**Pull request merged**